CHANGES.md

```diff
@@ -1,3 +1,42 @@
+# v0.23.0
+
+New Features:
+
+* Added [CloudStack](http://cloudstack.apache.org) provider support (thanks @syed; GH-558).
+* **❢ BREAKING CHANGE:** Added a more flexible benchmark configuration system
```

Comment:
Is this a breaking change because the benchmark_spec was expanded?  Just trying to understand what might break.